scripts/nginx/overseerr.sh

```diff
@@ -1,15 +1,38 @@
 #!/usr/bin/env bash
 
 cat > /etc/nginx/apps/overseerr.conf << EOF
-location /overseerr/ {
-  proxy_pass http://localhost:5055/;
+location ^~ /overseerr {
+    set \$app 'overseerr';
+    # Remove /overseerr path to pass to the app
+    rewrite ^/overseerr/?(.*)$ /\$1 break;
+    proxy_pass http://127.0.0.1:5055;  # NO TRAILING SLASH
+    # Redirect location headers
+    proxy_redirect ^ /\$app;
+    proxy_redirect /setup /\$app/setup;
+    proxy_redirect /login /\$app/login;
+    # Sub filters to replace hardcoded paths
+    proxy_set_header Accept-Encoding "";
+    sub_filter_once off;
+    sub_filter_types *;
+    sub_filter 'href="/"' 'href="/\$app"';
+    sub_filter 'href="/login"' 'href="/\$app/login"';
+    sub_filter 'href:"/"' 'href:"/\$app"';
+    sub_filter '/_next' '/\$app/_next';
+    sub_filter '/api/v1' '/\$app/api/v1';
+    sub_filter '/login/plex/loading' '/\$app/login/plex/loading';
+    sub_filter '/images/' '/\$app/images/';
+    sub_filter '/android-' '/\$app/android-';
+    sub_filter '/apple-' '/\$app/apple-';
+    sub_filter '/favicon' '/\$app/favicon';
+    sub_filter '/logo.png' '/\$app/logo.png';
+    sub_filter '/site.webmanifest' '/\$app/site.webmanifest';
 }
 EOF
```

Comment:
shell check is yelling about this; not entirely sure why.

Reply:
I think the shellcheck is failing because the container is failing to build, nothing with your code.